vortex-layout/Cargo.toml

```diff
@@ -21,18 +21,20 @@ bit-vec = { workspace = true }
 bytes = { workspace = true }
 exponential-decay-histogram = { workspace = true }
 flatbuffers = { workspace = true }
-futures = { workspace = true, features = ["alloc"] }
+flume = { workspace = true }
```

Comment:
Can this be optional behind a `threads` feature?

Reply:
so wdyt should be the default behavior?